### DISCUSSION TITLE
I have deleted the public organization, now i am unable to create it back neither anyone else.

### DISCUSSION THREAD

**Discussion by yaseen104** [community/community#146166] [npm]
I have deleted the organization in npm. Now neither i am able to create it neither anyone else.
Is there a cool down period? What needs to be done here? whom to contact and how to create the same organization again?

**Comment by Thyrail** [comment]
When you delete an organization on NPM, its name is typically reserved for a period to prevent immediate re-registration. This is to avoid confusion, abuse, or unauthorized usage of the same organization name.

## Steps to Address the Issue:

1. **Cool Down Period:**

- NPM enforces a cool-down period after an organization is deleted. During this time, neither you nor anyone else can recreate the organization with the same name. The exact duration of this period isn't publicly documented, so you may need to contact NPM support for confirmation.

2. **Contact NPM Support:**

- To expedite the process or inquire about reclaiming the organization, you should contact NPM support. Use their [Support Request Form](https://www.npmjs.com/support) and provide the following details:
  - Organization name.
  - Date of deletion.
  - Your NPM account username.
  - Reason for wanting to recreate the organization.

3. **Alternative Solutions:**

- If the organization name is critical, wait for the cool-down period to expire. In the meantime, consider using a placeholder organization name to continue your work.

4. **Prevent Future Issues:**

- If you decide to delete and recreate an organization again in the future, consider transferring the ownership or renaming it before deletion to avoid losing control of the name temporarily.

---

I recommend contacting NPM support directly, as they are best equipped to resolve this for you. If my answer helps you, I’d appreciate it if you could mark this discussion as closed. Thank you, and good luck!
